ocfl-java-api/src/main/java/io/ocfl/api/OcflFileRetriever.java

```diff
@@ -41,4 +42,17 @@ public interface OcflFileRetriever {
      * @return FixityCheckInputStream of the file's content
      */
     FixityCheckInputStream retrieveFile();
+
+    /**
+     * Returns an input stream of the file's content between the specified byte range. startPosition and endPosition
+     * may be null, depending on the underlying implementation, and the meaning of a null value is also implementation
+     * dependent.
+     *
+     * <p>The caller is responsible for closing the stream. The input stream is buffered.
+     *
+     * @param startPosition the byte offset in the file to start reading, inclusive
+     * @param endPosition the byte offset in the file to stop reading, inclusive
+     * @return a buffered input stream containing the specified file data
+     */
+    InputStream retrieveRange(Long startPosition, Long endPosition);
```

Comment:
@pwinckles  I would use the primitive datatype long instead of a Long object as parameter

Reply:
Can't because nulls are valid